src/transition.js

```diff
@@ -45,7 +46,8 @@ export default class RouteTransition {
       // on initial load, it gets caught in an infinite loop.
       const abortingOnLoad = !this.from.path && this.to.path === '/'
       if (!abortingOnLoad) {
-        this.router.replace(this.from.path || '/')
+        const path = this.from.path ? tryDecode(this.from.path, true) : '/'
```

Comment:
Can you add a little comment over here to point out why we need to call `tryDecode`?